docs/running-with-FIPS.md

```diff
@@ -1,50 +1,68 @@
 Using ziti-edge-tunnel in FIPS mode.
 ========================================================
 
+>[!NOTE]
+>A release binary linked to the system's OpenSSL v3 library is provided as a Linux package for Ubuntu 22.04 and later LTS
+>releases that are FIPS certified through Ubuntu Pro.
+
 This document describes how to build and run ziti-edge-tunnel in FIPS mode.
 
 Using FIPS mode requires the following:
-- dynamic linking against OpenSSL library
-- OpenSSL FIPS module
-- OpenSSL FIPS configuration
 
-## Dynamically linking against OpenSSL library
```

Comment:
This could use some restructuring. At the very least to mention that the zet binary from our packages for distros x, y, and z are already linked with dynamic openssl libraries, and all of the building stuff that follows is only needed if you a) aren't on distro x, y, or z, or b) don't want to use our packages for whatever reason.

Reply:
It's in a NOTE admonition at the top, but perhaps the placement could be improved.